showcase/src/main/kotlin/com/kevlar/showcase/data/repo/IntegrityRepository.kt

```diff
@@ -87,10 +100,9 @@ class IntegrityRepository @Inject constructor(
     )
 
 
-    private val g = HardcodedBase64EncodedFingerprint(
-        listOf(
-            ""
-        )
+    private val hardcodedBase64EncodedFingerprint = HardcodedBase64EncodedFingerprint(
+        sha1Base64SignatureFingerprints,
```

Comment:
On My device, these checks fail (both signature and fingerprint), because I don't have signing keys

Reply:
Will follow up, I can test with my keys